ansible/playbooks/adhoc/hotfix/roles/update_master/tasks/main.yml

```
@@ -56,7 +53,7 @@
   with_items:
   - atomic-openshift-master-api
   - atomic-openshift-master-controllers
-  when: "{{ update_needed == True and update_successful == True }}" 
+  when: "{{ update_needed == True and wanted_version_installed == True }}"
 
 - name: Ensure master services are running
   service:
```

Why are we ignoring errors here?

---

My thought was that if a master service failed to restart, I wanted to continue on to update the rest of the hosts, and add that master back to the ELB.

In retrospect, that doesn't make much sense for masters. I can remove that.

I think it makes more sense for nodes, where you might want to continue upgrading the rest of the nodes even though one node-restart failed.